File: Politicos_C++/prefeito.cpp
#include <iostream>
#include <stdio.h>
#include <stdlib.h>
#include "prefeito.h"
using namespace std;
Prefeito::Prefeito(string nome, string partido, string municipio, string estado){
    setNome(nome);
    setPartido(partido);
    setMunicipio(municipio);
    setEstado(estado);
    setFuncao("administrar o IPTU visando o melhor para a cidade\n");
}
Prefeito::~Prefeito(){
    setNome("");
    setPartido("");
    setEstado("");
    setMunicipio("");
}
void Prefeito::setMunicipio(string municipio){
    this->municipio = municipio;
}
string Prefeito::getMunicipio(){
    return this->municipio;
}
void Prefeito::apresentacao(){
Politico::apresentacao();
cout<<"Sou prefeito em "<<this->getNome()<<endl;
cout<<"Minha funcao é "<<this->getPartido()<<endl;
}

File: Politicos Python/Prefeito.py
from Politico import Politico
class Prefeito(Politico):
    def __init__(self,nome,partido,municipio,estado):
        Politico.__init__(self)
        self.setNome(nome)
        self.setPartido(partido)
        self.set_Municipio(municipio)
        self.setEstado(estado)
        self.setFuncao("Administrar o IPTU visando o melhor para a cidade")
        self.setSalario(1000)
    def set_Municipio(self,municipio):
        self.municipio = municipio
    def get_Municipio(self):
        return self.municipio
    def apresentacao(self):
        Politico.apresentacao(self)
        print(f'Sou prefeito em {self.get_Municipio()}/{self.getEstado()} e meu suposto salário é R$ {self.getSalario()}')
